FireboltNETSDK/Client/FireboltConnection.cs

```diff
@@ -170,7 +178,7 @@ public FireboltConnection(string connectionString)
         /// Initializes a new instance of <see cref="FireboltConnection"/> with the settings.
         /// </summary>
         /// <param name="stringBuilder">The connection string builder which will be used for building the connection settings.</param>
-        public FireboltConnection(FireboltConnectionStringBuilder stringBuilder)
+        private FireboltConnection(FireboltConnectionStringBuilder stringBuilder)
```

Comment:
I don't think this is correct. At least it's not consistent with our readme that suggests instantiating this class directly - https://github.com/firebolt-db/firebolt-net-sdk#creating-and-closing-a-connection

Reply:
FireboltConnection can be instantiated directly using constructor that accepts string, not the builder. The builder should become gory details of the implementation not exposed to the user.

Reply:
Ok, makes sense.